docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst

```diff
@@ -0,0 +1,267 @@
+and is aimed at improving the nutritional status or preventing the nutritional deterioration of the target population. One of the nutritional
+interventions advocated to address malnutrition among children is **lipid-based nutrient supplements (LNS)**. LNS are a family of
+products designed to deliver nutrients to vulnerable people. They are considered 'lipid-based' because most of the energy provided
+by these products is from lipids (fats). All LNS provide a range of vitamins and minerals, but unlike most other micronutrient supplements,
+LNS also provide *energy, protein and essential fatty acids*. LNS recipes can include a variety of ingredients, but typically have included vegetable fat, peanut or groundnut paste, milk powder and sugar. Based on the energy content, LNS can be small quantity (SQ LNS) providing ˜ 110 to 120 kcal/day (20 g dose), medium quantity (MQ LNS) providing ˜ 250 to 500 kcal/day (45 g to 90 g dose) or large-quantity (LQ LNS) providing
+more than 280 kcal/day (> 90 g dose).LNS are nutrient dense, require no cooking before use, and can be stored for months even in warm conditions. 
+[DAS_Cochrane_Review_2019]_
+
+:underline:`Alternative recipes and formulations, other than LNS`
+
+Alternative recipes and formulations, other than LNS, are currently
+being explored using cereals mixed with other ingredients, including
+whey, soy protein isolate, dried skimmed milk, and sesame,
+cashew and chickpea paste, among others. These are
+fortified with vitamins and minerals and are commonly called fortified
+blended foods (FBF). An example of a commonly used FBF
+is corn soy blend plus (CSB ++), which is a cooked blend of milled,
+heat-treated corn and soybeans that is fortified with a vitamin and
+mineral premix. Multiple micronutrient powders (MNP) are also an
+alternative way of providing micronutrients. These are single-dose
+packets of vitamins and minerals in powder form that can be sprinkled
+onto any ready to eat semi-solid food consumed at home,
+school or any other point of use. [DAS_Cochrane_Review_2019]_
+
+.. todo::
+
+   Add a general narrative overview of the intervention, including what it is, what outcomes it affects, if/how/when/where it has been used, etc.
+
+.. todo::
+
+  Fill out the following table with a list of known outcomes affected by the intervention, regardless of if they will be included in the simulation model or not, as it is important to recognize potential unmodeled effects of the intervention and note them as limitations as applicable.
+
+  The table below provides example entries for large scale food fortification with iron.
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note (ex: is this relationship direct or mediated?)
+  * - Hemoglobin concentration
+    - Increases population mean
+    - Yes
+    - 
+  * - Malaria
```

Comment:
This is a surprise to me, so I'm checking if you intend to include it

Reply:
ah, these are the examples from the template...I've left them in there and will revise as I build out the document.